lib/workers/repository/process/lookup/index.ts

```diff
@@ -569,6 +569,12 @@ export async function lookupUpdates(
 
     if (isReplacementRulesConfigured(config)) {
       addReplacementUpdateIfValid(res.updates, config);
+    } else if (dependency?.replacementName || dependency?.replacementVersion) {
```

Comment:
There's code in datasource which assumes that both properties must be set together. Wouldn't it be safer to do the same thing here, so turn the `||` into `&&`?

Reply:
Yes this makes more sense.